Add the SHARED doc page (#246)
rcc-uchicago/user-guide: docs/shared.md

# SHARED
<!-- From these links:
https://www.lib.uchicago.edu/about/news/uchicago-launches-shared-a-new-data-platform-for-research-collaboration-and-discovery/ -->

## What is SHARED?

The University of Chicago has received funding from the National Science Foundation (NSF) under Award No 2346746, to develop SHARED, the Secure Hub for Access, Reliability, and Exchange of Data. This new storage platform will advance research through robust capabilities for data storage, accessibility, sharing and integration with national and global data federations. It will provide critical storage capacity for research and supporting data science curricula development and education. For more

> information please see [the reference](https://dl.acm.org/doi/10.1145/3708035.3736042).

Reviewer: ```suggestion
information please see [this reference](https://dl.acm.org/doi/10.1145/3708035.3736042).
```

The SHARED platform will be essential for enabling partner faculty to meet federal requirements for data management and sharing. By supporting projects in fields ranging from cosmology and particle physics to linguistics and neuroscience, SHARED will help scientists store, access, and share massive datasets that drive discovery. Examples include data from dark matter searches, cosmic microwave background surveys, large-scale astrophysical simulations, high energy physics experiments, and research on language, learning, and memory. Specific projects include Cosmic Reionization on Computers, Function of Coordinated Hippocampal-cortical Activity in Learning, Grammatical Variation and Change in Sign Languages, How Children Use Gestures with Language, Kinetic Simulations of Astrophysical Plasmas, KOTO particle physics experiment probing rare Kaon decay, and, XENON underground physics experiment searching for dark matter using liquid Xenon detectors, and the South Pole telescope that studies the cosmic microwave background from Antarctica.

## Allocations

> The storage allocations for SHARED are prescribed by the grant. The group ownership of the project directories under `/shared` has been assigned to the main PIs of the SHARED grant. 

Reviewer: ```suggestion
The majority of the storage available in SHARED is reserved and for the projects and research groups that are part of the grant. Each project has a directory under `/shared` and the group ownership of these directories are initially assigned to the main PIs of each project. Requests for changes of ownership, creation of new groups, or questions about permissions can be submitted to shared@rcc.uchicago.edu. 
```

There are reservations of 50 TB will be reserved for educational, broader impact, and outreach activities, including open pedagogical data sets. In addition, an initial 50 TB will be reserved for open access data repositories for published research findings and the data that can be used to verify the published findings in accordance with FAIR principles. Allocation policies will be reviewed periodically by the SHARED executive committee.

## Access

> For the group members of the PIs of the grant, the SHARED storage system can be accessed via:

Reviewer: ```suggestion
There are different services and gateways for the members of the grant PIs to access the SHARED storage system, as listed below:
```

* Mount point in the Midway3 login nodes `/shared`
* Mount point in the Midway2 and DaLI login nodes `/shared`

> * Through gateway services such as the [Globus](globus/share-files.md) endpoint `UChicago RCC SHARED`

Reviewer: ```suggestion
* [Globus](globus/share-files.md) collection `UChicago RCC SHARED`
* [Samba](samba.md) share`\\shared-smb.rcc.uchicago.edu\shared` with username `ADLOCAL\cnetid`

Please note:

* The `/shared` mount points in the Midway systems is available only in the login nodes and cannot be mounted in the compute nodes as the SHARED filesystem is built for resiliency, data durability and stability, and not optimized for highly parallel I/O workloads.
* For security reasons the Samba share can only be accessed from the UChicago networks, i.e., campus network or cVPN.
```

Data can be transferred between `/shared` and other [storage spaces](storage/main.md) in the Midway ecosystem using command line interface tools such as `cp`, `scp`, `rsync` and `rclone`.

> 

Reviewer: ```suggestion
Project members can submit requests for additional gateway services and storage endpoints to the email shared@rcc.uchicago.edu. A member of the SHARED team will analyze the feasibility and contact the requestor for additional details. 
```
